src/main/java/org/broadinstitute/hellbender/utils/config/CustomBooleanConverter.java

```diff
@@ -0,0 +1,16 @@
+package org.broadinstitute.hellbender.utils.config;
+
+import org.aeonbits.owner.Converter;
+
+import java.lang.reflect.Method;
+
+/**
+ * Created by jonn on 7/21/17.
+ */
+public class CustomBooleanConverter implements Converter<Boolean> {
+
+    @Override
+    public Boolean convert(Method method, String input) {
+        return Boolean.getBoolean( input.trim() );
```

Comment:
Does this do what it's supposed to do?  Boolean.getBoolean(String) pulls a boolean value from a system property.  It seems like this should be Boolean.parseBoolean().  It should probably have a test.